addons/gitlab/requirements.txt

```diff
@@ -1,3 +1,3 @@
 cachecontrol==0.10.2
-python-gitlab==1.4.0
+python-gitlab==2.5.0
```

Comment:
I also laud the dependency upgrade, but were there any breaking changes between versions?

Reply:
Only changes needed for the update were [here](https://github.com/CenterForOpenScience/osf.io/pull/9602/files#diff-1aa3052bc89c301ee1df5a3836dcc327169d68a71f58f3724d1a9d61ab57418eR78) and [here](https://github.com/CenterForOpenScience/osf.io/pull/9602/files#diff-1aa3052bc89c301ee1df5a3836dcc327169d68a71f58f3724d1a9d61ab57418eR52)